pkg/defaults/defaults.go

```diff
@@ -387,8 +387,12 @@ func stepConfigsForBuild(config *api.ReleaseBuildConfiguration, jobSpec *api.Job
 		}
 	}
 
-	for i := range config.Tests {
-		buildSteps = append(buildSteps, api.StepConfiguration{TestStepConfiguration: &config.Tests[i]})
+	for _, test := range config.Tests {
+		// TODO remove when the migration is completed
+		if len(test.From) == 0 {
+			continue
+		}
+		buildSteps = append(buildSteps, api.StepConfiguration{TestStepConfiguration: &test})
```

Comment:
I am pretty sure taking the pointer to this `test` var won't work since it's reassigned in every loop iteration, which is why we were using `i` last time.

Reply:
Yeah, I edited too quickly, that surely won't work as it is.